lxd/daemon_storage.go

```diff
@@ -133,7 +133,7 @@ func daemonStorageMount(s *state.State) error {
 }
 
 func daemonStorageSplitVolume(volume string) (poolName string, volumeName string, err error) {
-	fields := strings.Split(volume, "/")
+	fields := strings.SplitN(volume, "/", 3)
```

Comment:
strings.Cut?

Reply:
I stuck with `strings.Split` because this allows catching bogus param like `<pool>/<volume>/<junk>`. However as you suggested doing more validation by splitting `daemonStorageValidate` into multiple functions, maybe that's OK to use `strings.Cut` and let the DB lookup deal with the bogus volume name `<volume>/<junk>`?

Reply:
Yeah let the db lookup handle it